src/eureka/S3_data_reduction/plots_s3.py

```diff
@@ -97,7 +133,7 @@ def image_and_background(data, meta, log, m):
     median = np.ma.median(subbg)
     std = np.ma.std(subbg)
     # Set bad pixels to plot as black
-    cmap = mpl.cm.get_cmap("plasma").copy()
+    cmap = plt.cm.plasma.copy()
```

Comment:
I don't think we need `.copy()` with your new implementation (only with get_cmap).

Reply:
`plt.cm.plasma` does return a static memory address (or so it seems), so if it get's called later I can't say with certainty that any changes to the cmap (e.g., bad colours/values) wouldn't end up affecting all the other places where that cmap was used